File: lab19/task5.py
class Car:
    def __init__(self, brand, model, year):
        self.brand = brand
        self.model = model
        self.year = year

    def display_details(self):
        print("Car Details:")
        print("Brand:", self.brand)
        print("Model:", self.model)
        print("Year:", self.year)

# Create an object and call the method
my_car = Car("Toyota", "Corolla", 2020)
my_car.display_details()

File: lab19/task5.js
class Car {
  constructor(brand, model, year) {
    this.brand = brand;
    this.model = model;
    this.year = year;
  }

  displayDetails() {
    console.log("Car Details:");
    console.log("Brand:", this.brand);
    console.log("Model:", this.model);
    console.log("Year:", this.year);
  }
}

// Create an object and call the method
const myCar = new Car("Toyota", "Corolla", 2020);
myCar.displayDetails();